readme.rst

```diff
@@ -262,3 +282,41 @@ Change log
 ----------
 
 See the `change log <https://github.com/canonical/sphinx-docs-starter-pack/wiki/Change-log>`_ for a list of relevant changes to the starter pack.
+
+Beta feature: Setup Script
```

Comment:
This position is too late and it also makes the README too long. I suggest we include it just before the 'Standalone documentation repository' heading with a new paragraph and include the information as a sub-page:

"The next two sections provide manual steps for enabling the starter pack for your repository. See [Setup script][??] for an automated method (currently in beta)."